src/ax_interface/mib.py

```diff
@@ -41,6 +42,13 @@ async def start(self):
 
                 # run the background update task
                 self.update_data()
+                self.redis_exception_happen = False
```

Comment:
>redis_exception_happen [](http://example.com/codeflow?start=21&length=22)

I am worried about the contract of member `redis_exception_happen` between multiple classes. How about define a new argument `reconnect=False` in `reinit_data()`, and keep `redis_exception_happen` as a private member only in this class.

Reply:
Fixed, move redis_exception_happen to local variable. also because the python method override issue, add new method reinit_connection() to MIBUpdater interface.